seacrowd/sea_datasets/tydiqa/tydiqa.py

```diff
@@ -0,0 +1,422 @@
+    in the world. It contains language phenomena that would not be found in English-only corpora. To provide a realistic
+    information-seeking task and avoid priming effects, questions are written by people who want to know the answer, but
+    don’t know the answer yet, (unlike SQuAD and its descendents) and the data is collected directly in each language
+    without the use of translation (unlike MLQA and XQuAD).
+    """
+
+_HOMEPAGE = "https://github.com/google-research-datasets/tydiqa"
+_LICENSE = Licenses.APACHE_2_0
+_HF_URL = "https://huggingface.co/datasets/tydiqa"
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
+
+_SOURCE_VERSION_P = "1.0.0"
+_SOURCE_VERSION_S = "1.1.0"
+_SEACROWD_VERSION = "0.0.0"
+
+_URL = "https://storage.googleapis.com/tydiqa/"
+_PRIMARY_URLS = {
+    "train": _URL + "v1.0/tydiqa-v1.0-train.jsonl.gz",
+    "dev": _URL + "v1.0/tydiqa-v1.0-dev.jsonl.gz",
+}
+_SECONDARY_URLS = {
+    "train": _URL + "v1.1/tydiqa-goldp-v1.1-train.json",
+    "dev": _URL + "v1.1/tydiqa-goldp-v1.1-dev.json",
+}
+
+_PRIMARY_DESP = """Passage selection task (SelectP): Given a list of the passages in the article, return either (a) the index of
+              the passage that answers the question or (b) NULL if no such passage exists.
+              Minimal answer span task (MinSpan): Given the full text of an article, return one of (a) the start and end
+              byte indices of the minimal span that completely answers the question; (b) YES or NO if the question requires
+              a yes/no answer and we can draw a conclusion from the passage; (c) NULL if it is not possible to produce a
+              minimal answer for this question."""
+
+_SECONDARY_DESP = """Gold passage task (GoldP): Given a passage that is guaranteed to contain the
+          answer, predict the single contiguous span of characters that answers the question. This is more similar to
+          existing reading comprehension datasets (as opposed to the information-seeking task outlined above).
+          This task is constructed with two goals in mind: (1) more directly comparing with prior work and (2) providing
+          a simplified way for researchers to use TyDi QA by providing compatibility with existing code for SQuAD 1.1,
+          XQuAD, and MLQA. Toward these goals, the gold passage task differs from the primary task in several ways:
+          only the gold answer passage is provided rather than the entire Wikipedia article;
+          unanswerable questions have been discarded, similar to MLQA and XQuAD;
+          we evaluate with the SQuAD 1.1 metrics like XQuAD; and
+          Thai and Japanese are removed since the lack of whitespace breaks some tools.
+          """
+
+
+class TydiqaDataset(datasets.GeneratorBasedBuilder):
+    """
+    TyDi QA is a question answering dataset covering 11 typologically diverse languages with 204K question-answer pairs.
```

Comment:
it seems this one is exactly the same as the `_DESCRIPTION` field. 
Would it be better to change this into a string indicating a main class of the data loader?
(e.g. `This is a main class of SEACrowd dataloader for ...`)